src/data/zcl_abapgit_data_config.clas.abap

```diff
@@ -24,20 +22,20 @@ ENDCLASS.
 
 
 
-CLASS ZCL_ABAPGIT_DATA_CONFIG IMPLEMENTATION.
+CLASS zcl_abapgit_data_config IMPLEMENTATION.
 
 
   METHOD dump.
 
-    DATA lo_ajson TYPE REF TO zcl_abapgit_ajson.
-    DATA lx_ajson TYPE REF TO zcx_abapgit_ajson_error.
-
+    DATA:
+      lo_ajson TYPE REF TO zcl_abapgit_ajson,
+      lx_ajson TYPE REF TO zcx_abapgit_ajson_error.
```

Comment:
the previous code follows https://github.com/SAP/styleguides/blob/main/clean-abap/CleanABAP.md#do-not-chain-up-front-declarations 🤷‍♂️  so someone will probably change it back again

Reply:
you can tell, I disagree with the rule ;-). the zig-zag and repeating of `DATA` makes it harder to parse with the eye and then there's PP to align this block automatically. if you prefer the "clean" style, i will change it. (should go on here too https://docs.abapgit.org/development/guidelines.html)

Reply:
changed back